change/@ni-nimble-components-d6d8517b-312c-467f-8b69-bdaa581e67fa.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
The shoelace design system just switched to using the native `summary` and `detail` elements in their accordion style component because by leveraging them it allowed the [native browser search feature](https://github.com/shoelace-style/shoelace/pull/1470) to search the contents.

If the FAST templates do not allow browser search into the collapsed accordions and are not leveraging the native `summary` + `details` components for the accordion that would be a strong reason to fork their template (maybe just for accordion-item) to have that capability.